pos_payment_terminal/pos_payment_terminal.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data>
-
-<template id="assets_backend" name="point_of_sale assets" inherit_id="web.assets_backend">
-    <xpath expr="." position="inside">
-        <script type="text/javascript" src="/pos_payment_terminal/static/src/js/pos_payment_terminal.js"></script>
```

Comment:
Hi pedro, it is just refactoring. the include is here  :  https://github.com/OCA/pos/pull/105/files#diff-23a25d4fe2ebc9a4291377ba6c07a1aeR7